app/views/owned_tag_sets/_internal_tag_set_fields.html.erb

```diff
@@ -1,5 +1,5 @@
 <% TagSet::TAG_TYPES.each do |tag_type| %>
-  <fieldset class="tagset listbox group">
+  <fieldset class="tagset listbox group"l id="fieldset-<%= tag_type %>">
```

Comment:
There's a typo in here -- a stray l before the id attribute. 

However, it would actually be great if you could add the tag type to the classes (i.e. `<fieldset class="<%= tag_type %> tagset listbox group">`) instead of adding an id, because a class we could use for both tests and styling, while we'd only ever use an id for tests. I'm not sure if that's possible, though, since I don't see the reference to the id in these tests -- is this part of the prep work for additional tests?